winter/django.py

```diff
@@ -71,28 +75,45 @@ class WinterView(rest_framework.views.APIView):
 def _create_dispatch_function(controller, controller_method: ControllerMethod):
     def dispatch(winter_view, request: Request, **path_variables):
         try:
-            arguments = arguments_resolver.resolve_arguments(controller_method, request, path_variables)
-            result = controller_method.func(controller, **arguments)
-            if isinstance(result, django.http.HttpResponse):
-                return result
-            if isinstance(result, ResponseEntity):
-                body = result.entity
-                status_code = result.status_code
-            else:
-                body = result
-                status_code = get_default_response_status(controller_method)
-            output_processor = get_output_processor(controller_method.func, body)
-            if output_processor is not None:
-                body = output_processor.process_output(body, request)
-            if isinstance(body, django.http.HttpResponse):
-                return body
-            response = rest_framework.response.Response(body, status=status_code)
+            return _call_controller_method(controller, controller_method, path_variables, request)
         except WinterException as exception:
-            response = handle_winter_exception(exception)
-        return response
+            return handle_winter_exception(exception)
+
     return dispatch
 
 
+def _call_controller_method(controller, controller_method: ControllerMethod, path_variables: Dict, request: Request):
+    arguments = arguments_resolver.resolve_arguments(controller_method, request, path_variables)
+    try:
+        result = controller_method.func(controller, **arguments)
+        return _convert_result_to_http_response(request, result, controller_method.func)
+    except Exception as e:
```

Comment:
You can use 
`
except tuple(get_throws(func)) as e

`
And delete is_expected_to_throw